WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -223,19 +223,20 @@
         [
           t('.events'),
           true,
-          @competition.events.select(&:official?),
-          Event.all_official,
+          @competition.event_picker_events,
```

Comment:
@pingskills I see you edited the comment and changed your mind, but actually I think that we could do this. The only thing we do with selected_events is `selected_events.include?(event)`, so if it contains more events it shouldn't matter at all.